be/src/common/status.cpp

```diff
@@ -50,18 +51,20 @@ Status::Status(const PStatus& s) : _state(nullptr) {
     TStatusCode::type code = (TStatusCode::type)s.status_code();
     if (code != TStatusCode::OK) {
         if (s.error_msgs_size() == 0) {
-            _state = assemble_state(code, Slice(), 1, Slice());
+            _state = assemble_state(code, Slice(), -1, Slice());
```

Comment:
keep 1 as default code? Because minus number is not good for serialization.

Reply:
Will precise_code often use `POSIX errno` to indicate detailed error codes in the future? 
If it is, what we need to pay attention to is: In POSIX code, `1` is only used to refer
to `EPERM`(`Operation not permitted`), and **`-1` is more general**.
In addition, the type of this precise_code is `int16_t`, a signed integer.

And what is the "serialization problem"?

Reply:
This has no relationship with posix code. For user reference, positive numbers are more convenient than negative numbers.
When doing serialization, positive number can be encoded into smaller space, for example varint encode.

Reply:
OK, change it to `1`